lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -112,6 +131,34 @@ def on_class_node_leave(node)
 
       private
 
+      sig { returns(T.nilable(String)) }
+      def migration_version
+        File.basename(T.must(@path)).split("_").first
+      end
+
+      sig { params(node: Prism::Node, name: String, command: String).void }
+      def add_migrate_code_lens(node, name:, command:)
+        return unless @path
+
+        arguments = [
```

Comment:
Does the command in the VS Code extension need the location information? If it's just using the command to run the migration, then we can remove the location information here and just return command.